src/parameterizations/lateral/MOM_hor_visc.F90

```diff
@@ -452,12 +455,13 @@ subroutine horizontal_viscosity(u, v, h, diffu, diffv, MEKE, VarMix, G, GV, US,
                  (CS%bound_Kh .and. .not.CS%better_bound_Kh)
 
   if (CS%use_GME) then
+
     do j=Jsq-1,Jeq+2 ; do i=Isq-1,Ieq+2
       boundary_mask_h(i,j) = (G%mask2dCu(I,j) * G%mask2dCv(i,J) * G%mask2dCu(I-1,j) * G%mask2dCv(i,J-1))
     enddo ; enddo
 
-    do J=js-2,Jeq+1 ; do I=is-2,Ieq+1
-      boundary_mask_q(I,J) = (G%mask2dCv(i,J) * G%mask2dCv(i+1,J) * G%mask2dCu(I,j) * G%mask2dCu(I,j-1))
+    do J=Jsq-2,Jeq+1 ; do I=Isq-2,Ieq+1
```

Comment:
I suspect that these loop bounds should have symmetric halos, i.e., they should be `do J=Jsq-1,Jeq+1 ; do I=Isq-1,Ieq+1`.  I agree with the corrections to the index offsets on the next line.